lib/models/project.js

```diff
@@ -132,7 +132,8 @@ class Project {
       // Blueprint tests set this flag.
       return true;
     }
-    for (let depName in this.dependencies()) {
+    const excludeDevDeps = true;
+    for (let depName in this.dependencies(this.pkg, excludeDevDeps)) {
       try {
         this.resolveSync(depName);
```

Comment:
Can you tweak this to use the `/package` thing I mentioned in another comment?  I think that is the only real blocker for this PR...

Reply:
I'll give that a go. Thanks